task1/index.js

```diff
@@ -0,0 +1,117 @@
+const notes = [
```

Comment:
Let's rename this constant using uppercase. When we declaring a variable that contains a collection of items or plain value (string, number), without any getters/setters/calculations -> uppercase
constant with getters/setters/calculations -> camelcase
`const NOTES =`